framework/templates/ecl-templates-search/ecl-templates-search.twig

```diff
@@ -10,77 +10,128 @@
-          {% block form %}
-            {% embed '@ec-europa/ecl-forms-form-groups' %}
-              {% block form_group %}
-                {% include '@ec-europa/ecl-forms-labels' with { 'label': 'Source', 'for_attribute': 'search-option-source' } %}
-                {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'source', 'id': 'search-option-source' } %}
-                  {% block options %}
-                    <option value="all">All sources</option>
-                    <option value="dg-trade" selected>DG Trade</option>
-                  {% endblock %}
-                {% endembed %}
-              {% endblock %}
-            {% endembed %}
-            {% embed '@ec-europa/ecl-forms-form-groups' %}
-              {% block form_group %}
-                {% include '@ec-europa/ecl-forms-labels' with { 'label': 'Date', 'for_attribute': 'search-option-date' } %}
-                {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'date', 'id': 'search-option-date' } %}
-                  {% block options %}
-                    <option value="all">All dates</option>
-                    <option value="today">Today</option>
-                    <option value="last_week">Last week</option>
-                    <option value="last_year">Last year</option>
-                  {% endblock %}
-                {% endembed %}
-              {% endblock %}
-            {% endembed %}
-            {% embed '@ec-europa/ecl-forms-form-groups' %}
-              {% block form_group %}
-                {% include '@ec-europa/ecl-forms-labels' with { 'label': 'Language', 'for_attribute': 'search-option-language' } %}
-                {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'language', 'id': 'search-option-language' } %}
-                  {% block options %}
-                    <option value="en">English</option>
-                    <option value="de">Deutsch</option>
-                    <option value="fr">Français</option>
-                  {% endblock %}
-                {% endembed %}
-              {% endblock %}
-            {% endembed %}
-            {% embed '@ec-europa/ecl-forms-form-groups' %}
-              {% block form_group %}
-                {% include '@ec-europa/ecl-forms-labels' with { 'label': 'File Format', 'for_attribute': 'search-option-file-format' } %}
-                {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'file-format', 'id': 'search-option-file-format' } %}
-                  {% block options %}
-                    <option value="all">All formats</option>
-                    <option value="doc">Doc</option>
-                    <option value="pdf">PDF</option>
-                    <option value="web">Web</option>
-                  {% endblock %}
+      <div class="ecl-u-d-flex ecl-u-d-lg-none ecl-u-justify-content-between ecl-u-align-items-baseline">
```

Comment:
These are in a light gray background in `General Listing Mobile Filters Displayed.png`